consensus/src/lib.rs

```diff
@@ -226,22 +227,47 @@ impl Consensus {
             .rev()
             .step_by(2)
         {
+            // Gets the strong edges from the vertex in round r+1 with the same id as the leader id
+            let votes = state
+                .dag
+                .get(&(r + 1))
+                .expect("vertices at round r+1")
```

Comment:
It is better to be explicit on error message (what is the problem in this case?)